### DISCUSSION TITLE
File association, default app for PDF, EPUB, MOBI, CBZ..

### DISCUSSION THREAD

**Discussion by stepger** [sumatrapdfreader/sumatrapdf#4314] [Ideas]
Hi,
it would be cool to have quick **tick** for 
file association that will set *SumatraPDF*
a default reader for
- PDF, EPUB, MOBI, CBZ, CBR, FB2, CHM, XPS, DjVu

Thanks

**Comment by kjk** [comment]
We do register for those in the installer. Do you mean in some other context?

**Reply by stepger** [reply]
Some apps like mpc-hc/be, and some picture viewers do this for multiple media formats, and 
it works on my win10.

Not a bib issue, just a suggestion, and 
we know how MS makes it harder for other apps to become a "default". 

thanks

**Comment by kjk** [comment]
To expand on that: all we can do on modern Windows is to register as an option.

Software cannot set itself as a default reader, it has to be explicit user action (either via settings app or right-click on a file in explorer). See https://www.sumatrapdfreader.org/docs/Set-as-default-PDF-viewer
